mlflow/store/model_registry/file_store.py

```diff
@@ -801,6 +824,61 @@ def delete_model_version_tag(self, name, version, key):
             updated_time = get_current_time_millis()
             self._update_registered_model_last_updated_time(name, updated_time)
 
+    def _get_registered_model_alias_path(self, name, alias):
+        _validate_model_name(name)
+        _validate_model_alias_name(alias)
+        registered_model_path = self._get_registered_model_path(name)
+        if not exists(registered_model_path):
+            raise MlflowException(
+                f"Registered Model with name={name} not found",
+                RESOURCE_DOES_NOT_EXIST,
+            )
+        return os.path.join(
+            registered_model_path, FileStore.REGISTERED_MODELS_ALIASES_FOLDER_NAME, alias
+        )
+
+    def set_registered_model_alias(self, name, alias, version):
+        """
+        Set a registered model alias pointing to a model version.
+
+        :param name: Registered model name.
+        :param alias: Name of the alias.
+        :param version: Registered model version number.
+        :return: None
+        """
+        alias_path = self._get_registered_model_alias_path(name, alias)
+        make_containing_dirs(alias_path)
+        write_to(alias_path, self._writeable_value(version))
+        updated_time = get_current_time_millis()
+        self._update_registered_model_last_updated_time(name, updated_time)
+
+    def delete_registered_model_alias(self, name, alias):
```

Comment:
What should happen if the associated registered model or version is deleted?

Reply:
Shouldn't the alias be automatically removed when the model is deleted since it's a subdirectory in the registered model directory? And we don't necessarily want the alias to be deleted when the version is deleted since it's a property on the model

Reply:
> Shouldn't the alias be automatically removed when the model is deleted since it's a subdirectory in the registered model directory

Makes sense

> we don't necessarily want the alias to be deleted when the version is deleted since it's a property on the model

Got it!